webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/SystemVariableService.scala

```diff
@@ -356,6 +356,7 @@ class SystemVariableServiceImpl(
       val varManagedNodes      = systemVariableSpecService.get("MANAGED_NODES_NAME" ).toVariable(nodesWithCFEKey.map(_.hostname))
       val varManagedNodesId    = systemVariableSpecService.get("MANAGED_NODES_ID"   ).toVariable(nodesWithCFEKey.map(_.id.value))
       val varManagedNodesKey   = systemVariableSpecService.get("MANAGED_NODES_KEY"  ).toVariable(nodesWithCFEKey.map(_.securityTokenHash))
+      val varPolicyServerKey   = systemVariableSpecService.get("POLICY_SERVER_KEY" ).toVariable(Seq(nodeInfo.securityTokenHash))
```

Comment:
it should have been the server securityTokenHash

Reply:
It is on the case where the node is a policy server. Do you mean it should have be the node's policy server?